website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -138,9 +138,28 @@ check "health_check" {
 
```

Comment:
it's not in the context of this PR so can't make a suggestion, but let's update the first section of the CV section to be this and remove the new paragraphs further down. i think adding a section about this preventing false positives will help ensure people dont gloss over it. is false positive the correct way to refer to this?

```
Continuous validation regularly verifies whether your configuration’s custom assertions continue to pass, validating your infrastructure. For example, you can monitor whether your website returns an expected status code, or whether an API gateway certificate is valid. Identifying failed assertions helps you resolve the failure and prevent errors during your next time Terraform operation.

Continuous validation evaluates preconditions, postconditions, and check blocks as part of an assessment, but we recommend using [check blocks](https://developer.hashicorp.com/terraform/language/checks) for post-apply monitoring. Use check blocks to create custom rules to validate your infrastructure's resources, data sources, and outputs.

### Preventing false positives

Health assessments create a speculative plan to access the current state of your infrastructure. Terraform evaluates any check blocks in your configuration as the last step of creating the speculative plan. If your configuration relies on data sources and the values queried by a data source change between the time of your last run and the assessment, the speculative plan will include those changes. Terraform Cloud will not modify your infrastructure as part of an assessment, but it can use those updated values to evaluate checks. This may lead to false positive results for alerts since your infrastructure did not yet change.

To ensure your checks evaluate the current state of your configuration instead of against a possible future change, use nested data sources that query your actual resource configuration, rather than a computed latest value. 
Refer to the [AMI image scenario](#asserting-up-to-date-amis-for-compute-instances) below for an example. 
```